src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Persons/PersonsControllerNegativeTests.cs

```diff
@@ -39,5 +39,91 @@ public async Task CreateSavedFilter_AsHacker_ShouldReturnForbidden_WhenPermissio
                 true,
                 HttpStatusCode.Forbidden);
         #endregion
+
+        #region GetSavedFiltersInProject
+        [TestMethod]
+        public async Task GetSavedFiltersInProject_AsAnonymous_ShouldReturnUnauthorized()
+            => await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Anonymous,
+                TestFactory.UnknownPlant,
+                null,
+                HttpStatusCode.Unauthorized);
+
+        [TestMethod]
+        public async Task GetSavedFiltersInProject_AsHacker_ShouldReturnBadRequest_WhenUnknownPlant()
+            => await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Hacker,
+                TestFactory.UnknownPlant,
+                null,
+                HttpStatusCode.BadRequest,
+                "is not a valid plant");
+
+        [TestMethod]
+        public async Task GetSavedFiltersInProject_AsViewer_ShouldReturnBadRequest_WhenUnknownProject() =>
+            await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                "12345",
+                HttpStatusCode.BadRequest);
+
+        [TestMethod]
+        public async Task GetSavedFiltersInProject_AsHacker_ShouldReturnForbidden_WhenPermissionMissing()
+            => await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Hacker,
+                TestFactory.PlantWithAccess,
+                null,
+                HttpStatusCode.Forbidden);
+        #endregion
+
+        #region UpdateSavedFilter
+        [TestMethod]
+        public async Task UpdateSavedFilter_AsAnonymous_ShouldReturnUnauthorized()
+            => await PersonsControllerTestsHelper.UpdateSavedFilter(
+                UserType.Anonymous,
+                TestFactory.UnknownPlant,
+                "new title",
+                "new criteria",
+                false,
+                "rowVersion",
```

Comment:
use a valid RowVersion ... this to be sure that the validation of an invalid rowversion mess up the issue under test. Same for tests below